pkg/controllers/readiness/wellknown_ready_controller.go

```diff
@@ -185,7 +185,7 @@ func (c *wellKnownReadyController) isWellknownEndpointsReady(spec *operatorv1.Op
 		return nil
 	}
 	if expectedMinNumber := getExpectedMinimumNumberOfMasters(spec); len(ips) < expectedMinNumber {
-		return fmt.Errorf("need at least %d kube-apiservers, got %d", expectedMinNumber, len(ips))
+		return fmt.Errorf("need at least %d kube-apiservers, got %d (see kube-apiserver operator for more information)", expectedMinNumber, len(ips))
```

Comment:
isn't this not only about existance, but about kube-apiservers all being up-to-date? Or is that error string somewhere else?

Reply:
```
waiting for at least %d kube-apiservers, kube-apiserver operators reports only %d (see installer logs and check why not all master nodes are up)
```